src/Aspire.Hosting.Azure.SignalR/AzureSignalRResource.cs

```diff
@@ -23,5 +31,7 @@ public class AzureSignalRResource(string name, Action<AzureResourceInfrastructur
     /// Gets the connection string template for the manifest for Azure SignalR.
     /// </summary>
     public ReferenceExpression ConnectionStringExpression =>
-        ReferenceExpression.Create($"Endpoint=https://{HostName};AuthType=azure");
+        IsEmulator
+        ? ReferenceExpression.Create($"Endpoint={EmulatorEndpoint.Property(EndpointProperty.Scheme)}://{EmulatorEndpoint.Property(EndpointProperty.Host)}:{EmulatorEndpoint.Property(EndpointProperty.Port)};AccessKey=ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789ABCDEFGH;Version=1.0;")
```

Comment:
could we update the emulator to not dependent on "AccessKey" so that we don't need to have the AccessKey in code although it is fake

Reply:
Revert the comment. Emulator could be working in an isolated environment, so Microsoft Entra ID should not be a dependency when using Emulator. AccessKey is a best choice for existing SDKs to work seamlessly.